src/pci.rs

```diff
@@ -136,6 +136,8 @@ impl Cap {
             unsafe { access.read(addr.address, addr.offset + 12) },
         ];
 
+        let data: [u32; 4] = data.map(u32::from_le);
+
         let this = unsafe { mem::transmute::<[u32; 4], Self>(data) };
```

Comment:
```suggestion
```

Reply:
That line is necessary to convert the raw data read to the capabilities structure, it cannot simply be dropped